tutorials/CMakeLists.txt

```diff
@@ -251,16 +259,17 @@ if(root7)
   if(MSVC AND NOT win_broken_tests)
     list(APPEND root7_veto v7/box.cxx v7/draw*.cxx )
   endif()
-endif()
-
-if(NOT root7)
-  set(root7_veto v7/ntuple/ntpl001_staff.C
-                 v7/ntuple/ntpl002_vector.C
-                 v7/ntuple/ntpl003_lhcbOpenData.C
-                 v7/ntuple/ntpl004_dimuon.C
-                 v7/ntuple/ntpl005_introspection.C)
-elseif(NOT dataframe)
-  set(root7_veto v7/ntuple/ntpl004_dimuon.C)
+  if(NOT dataframe)
+    set(root7_veto v7/ntuple/ntpl004_dimuon.C)
+  endif()
+else()
+  #  set(root7_veto v7/ntuple/ntpl001_staff.C
```

Comment:
Why do you want to keep the outcommented lines?

Reply:
That was a test. Given that the line below works, I can remove, indeed.